packages/frontend/src/components/admin/tabs/UsersTab.tsx

```
@@ -0,0 +1,174 @@
+};
+
+type UsersTabProps = {
+  data: User[];
+  onAdd: (user: Omit<User, 'id'>) => void;
+  onEdit: (user: User) => void;
+  onDelete: (user: User) => void;
+};
+
+export function UsersTab({ data, onAdd, onEdit, onDelete }: UsersTabProps) {
+  const [sheetOpen, setSheetOpen] = React.useState(false);
+  const [sheetMode, setSheetMode] = React.useState<'add' | 'edit'>('add');
```

ESLint

[error] 'sheetMode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)